web/src/views/source/bottomPanel.ts

```diff
@@ -113,6 +110,28 @@ class PanelElement extends LitElement {
     `
   }
 
+  private renderFormatSelect = () => {
+    const changeEvent = (e: Event) =>
+      (this.sourceView.format = (e.target as HTMLSelectElement).value)
```

Comment:
Entirely an opinion: does anyone like the parenthesis style function body? For readability I believe that block style functions offer more clarity.

Reply:
I'm not a fan of the readability of arrow functions generally but agree the block style is slightly less worse than parentheses.